charts/grafana/README.md

```diff
@@ -396,6 +396,12 @@ dashboards:
         value: Loki
     local-dashboard:
       url: https://raw.githubusercontent.com/user/repository/master/dashboards/dashboard.json
+      # default: -skf
+      # -s  - silent mode
+      # -k  - allow insecure (eg: non-TLS) connections
+      # -f  - fail fast
+      # -L  - follow HTTP redirects
+      curlOptions: -Lf
```

Comment:
```suggestion
      curlOptions: -skf
```

Reply:
Here I am showing how to change it away from the default value.

If I replace the default value with the default value value (made explicit), then somebody might think that `curlOptions` is required.

Let me know if you insist on your way and I'll change it even though I disagree.